crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -2074,8 +2073,7 @@ async fn custom_operation_timeout_sigkill() {
     }
     trap handle_term SIGTERM
     for i in {1..50}
-    do
-        echo "main $i"
```

Comment:
Why this has been removed?

The point was to check that operation script output is actually captured.

Reply:
In some cases, this log message is printed once, and sometimes it is printed multiple times before the signal is handled and the control is passed to the handler. So, because of this situation, it's hard to assert the whole output, because of the variable number of output logs from the main thread. This is the reason, I removed this log and just slept till the signal arrived and is being processed.